src/libs/auth.ts

```diff
@@ -42,27 +40,31 @@ export const getOidcConfig = () => {
     metadata,
     prompt: 'consent login',
     scope: 'openid email profile',
-    loadUserInfo: isGoogleAuthority(),
     stateStore: new WebStorageStateStore({ store: getLocalStorage() }),
     userStore: new WebStorageStateStore({ store: getLocalStorage() }),
     automaticSilentRenew: true,
-    // Leo's setCookie interval is currently 5 min, set refresh auth then 5 min 30 seconds to gurantee that setCookie's token won't expire between 2 setCookie api calls
+    // Leo's setCookie interval is currently 5 min, set refresh auth then 5 min 30 seconds to guarantee that setCookie's token won't expire between 2 setCookie api calls
     accessTokenExpiringNotificationTimeInSeconds: 330,
     includeIdTokenInSilentRenew: true,
     extraQueryParams: { access_type: 'offline' },
+    redirect_uri: '', // this field is not being used currently, but is expected from UserManager
   };
 };
 
-const isGoogleAuthority = () => {
-  return _.startsWith('https://accounts.google.com', authStore.get().oidcConfig.authorityEndpoint);
-};
-
 const getAuthInstance = () => {
   return authStore.get().authContext;
 };
 
 export const signOut = () => {
   // TODO: invalidate runtime cookies https://broadworkbench.atlassian.net/browse/IA-3498
+  const sessionEndTime: number = Date.now();
+  const tokenMetadata: any = authStore.get().authTokenMetadata;
```

Comment:
```suggestion
  const tokenMetadata = authStore.get().authTokenMetadata;
```

Let's avoid explicit `any` as much as possible.